.github/ISSUE_TEMPLATE/module_proposal_bicep_child.yml

```diff
@@ -0,0 +1,228 @@
+name: New Bicep Child Module Proposal 🗒️
+description: Want a new child module of a Bicep resource module to exist? Let us know!
+title: "[Bicep Child Module Proposal]: `BICEP_CHILD_MODULE_NAME_BETWEEN_BACKTICKS`"
+labels: ["Type: New Module Proposal :bulb:", "Needs: Triage :mag:", "Language: Bicep :muscle:", "Class: Child Module :package:"]
+projects: ["Azure/529"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for proposing a new Bicep child module! Please create a separate proposal for each child module.
+
+        This issue will automatically assigned to the owner of the parent module unless the module is currently orphaned.
+  - type: checkboxes
+    id: existing-checks
+    attributes:
+      label: Check for previous/existing GitHub issues/module proposals
+      description: By submitting this issue, you confirm that you have [searched for previous/existing GitHub issues/module proposals](https://github.com/Azure/Azure-Verified-Modules/issues?q=is%3Aissue+sort%3Aupdated-desc+) to avoid creating a duplicate.
+      options:
+        - label: I have checked for previous/existing GitHub issues/module proposals.
+          required: true
+  - type: checkboxes
+    id: existing-module-from-elsewhere
+    attributes:
+      label: Check this module doesn't already exist in the module indexes
+      description: By submitting this issue, you confirm that you have checked the [Bicep](https://azure.github.io/Azure-Verified-Modules/indexes/bicep/) module indexes for the module you are proposing to ensure it is not already available/being worked on.
+      options:
+        - label: I have checked for that this module doesn't already exist in the module indexes.
+          required: true
+  - type: dropdown
+    id: parentmodule
+    attributes:
+      label: Parent module?
+      description: Which parent module is this the child of?
+      options:
```

Comment:
Until automating this list update, should we allow a custom field?

Reply:
That can potentially lead to more efforts on the administration side to find out what the proposing party meant (think if typos, missing info, etc.). I'd vote for leaving this as is for now, and trying to catch up with the automation part as quick as possible. Until then, if a parent module is not in the list, people can ask me to add it to the list. If we find out that the standard input field would work better, I'm happy to change it back later.